builder/js/ncf.js

```diff
@@ -431,10 +432,42 @@ $scope.groupMethodsByCategory = function () {
       // Select the technique, by using angular.copy to have different objects
       $scope.selectedTechnique=angular.copy(technique);
       $scope.originalTechnique=angular.copy($scope.selectedTechnique);
+
       $scope.$broadcast('endSaving');
     }
   };
 
+  $scope.updateItemLocalStorage = function(item, oldTechnique, newTechnique){
+    if(oldTechnique && !newTechnique){
```

Comment:
Is it useful to compare on "OldTechnique?" I think it can be done on newTechnique Only don't you think ?